test/constants.js

```diff
@@ -101,7 +101,6 @@ const Contract = {
   ExchangeRateMock: 'ExchangeRateMock',
   PrngSystemContract: 'PrngSystemContract',
   Concatenation: 'Concatenation',
-  Errors: 'Errors',
```

Comment:
This was already there so no need to remove it right?
It should conflict with other PRs